src/api/app/models/issue_tracker.rb

```diff
@@ -32,6 +33,14 @@ def write_to_backend
     self.issues_updated ||= Time.now
   end
 
+  def verify_issue_name!(name)
```

Comment:
Issue is for all kind of trackers, so I would write it like this:

# then Issue model.is_valid_name? will return a boolean like value
  self.is_valid_name?(tracker, name)
    return Regexp.new(/^\d\d\d\d-\d+$/).match(name) if tracker.kind == "cve"
    true
  end

Note: I would not use a one liner here like 
   tracker.kind != "cve" || Regexp.new(/^\d\d\d\d-\d+$/).match(name)

since it is harder to read and to extend.

# Line 76 of patchinfo.rb can be like this
# please change one char variable names to something easy to understand, here I changed 'i' to 'issue'
raise IssueTracker::InvalidIssueName.new "The issue name is not supported: #{issue['id']}" unless Issue.is_valid_name?(self, issue['id'])